mscore/textpalette.cpp

```diff
@@ -612,6 +612,14 @@ TextPalette::TextPalette(QWidget* parent)
 //   populateCommon
 //---------------------------------------------------------
 
+int unicodeAccidentals[] = { //better size and alignment, so put these first
+      0x266d,    // flat
+      0x266e,    // natural
+      0x266f,    // sharp
+      0x1d12b,   // double flat
```

Comment:
It certainly is the reason why I haven't put them into the handbook or the translations